XRTK-Core/Packages/com.xrtk.core/Interfaces/InputSystem/IMixedRealityInputSystem.cs

```diff
@@ -203,6 +204,14 @@ public interface IMixedRealityInputSystem : IMixedRealityEventSystem
         /// <param name="position"></param>
         void RaiseSourcePoseChanged(IMixedRealityInputSource source, IMixedRealityController controller, MixedRealityPose position);
 
+        /// <summary>
+        /// Raise the event that the hand data for a input source has changed.
+        /// </summary>
+        /// <param name="source">The input source.</param>
+        /// <param name="handedness">The handedness of the input source.</param>
+        /// <param name="handData">The new hand data.</param>
+        void RaiseHandDataInputChanged(IMixedRealityInputSource source, Handedness handedness, HandData handData);
```

Comment:
What exactly is contained in the `HandData`?

Do we actually need a new event for this?

Reply:
It contains the hand joint poses and hand mesh information. Event is needed for controller visualizers to properly visualize.

Reply:
I honestly don't think we need this event to be so specific.

Reply:
Going to need some more explaination here what the issue with this event is as well. I don't see anything existing in place to communicate this sort of data.

Reply:
I think this change is fine after seeing the input data passed through